### DISCUSSION TITLE
Example of DF widget?

### DISCUSSION THREAD

**Discussion by ghost** [qtile/qtile#2893] [Q&A] [answered]
A simple example of DF widget.

**Comment by elParaguayo** [comment, marked as answer]
What problem are you having? 

The docs tell you how to configure it: http://docs.qtile.org/en/latest/manual/ref/widgets.html#df

Also note:
> By default the widget only displays if the space is less than warn_space.

**Reply by ghost** [reply]
I want the widget to show unused space but when i format it displays nothing.

**Reply by elParaguayo** [reply]
Did you read my post? 

As the docs say, by default, the space is only shown when the space is less than the warning amount. 

If you want it to show all the time you can change the `visible_on_warn` setting to `False`. All the available settings are listed on the page I linked. 

If it doesn't work, let me know.

**Reply by ghost** [reply]
Thank You it worked!!

**Reply by elParaguayo** [reply]
Great. The docs should tell you most things you need to know about the widgets but if you think there's anything missing, let us know.

**Reply by ghost** [reply]
Okay
